test/verify/check-users

```diff
@@ -570,6 +578,70 @@ class TestAccounts(MachineCase):
         self.allow_journal_messages(".*required to change your password immediately.*")
         self.allow_journal_messages(".*user account or password has expired.*")
 
+    def testCustomUserProperties(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/users")
+
+        # Test custom home directory
+        custom_dir_path = "/home/mycustomdir"
+        createUser(
+            browser=b,
+            user_name="jussi",
+            real_name="Jussi Junior",
+            password=good_password,
+            locked=False,
+            custom_home_dir=custom_dir_path,
+            force_password_change=False,
+            verify_created=True,
+        )
+
+        b.go("#/jussi")
+        b.wait_text("#account-home-dir", custom_dir_path)
+        m.execute(f"test -d {custom_dir_path}")
+        m.execute("! test -d /home/jussi")
+
+        b.go("/users")
+        # Check error handling
+        createUser(
+            browser=b,
+            user_name="Failer",
+            real_name="Failer Failerson",
+            password=good_password,
+            locked=False,
+            custom_home_dir="",
+            force_password_change=False,
+            verify_created=False,
+        )
+        b.wait_visible("#accounts-create-dialog")
+        b.wait_in_text("#accounts-create-user-home-dir-helper", "No home directory")
```

Comment:
As I said above, I find this a bit questionable. Empty should just be the default.

However, there are some realistic error conditions that you could check: Try specifying an existing file as home dir, like `/dev/null` even, or one which cannot be chowned, like `/proc/1`  :smiling_imp:

Reply:
> Try specifying an existing file as home dir, like /dev/null even, or one which cannot be chowned, like /proc/1 smiling_imp

That doesn't seem to stop useradd. It just show warning, but still creates user anyway with home dir as /proc/1:

```
useradd: warning: the home directory /proc/1 already exists.
useradd: Not copying any file from skel directory into it.
```

`Simon:x:1002:1002::/proc/1:/bin/bash`

Reply:
Fixed